src/components/SearchResults.tsx

```diff
@@ -1,18 +1,55 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
 import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import LifeEventPicker from './LifeEventPicker';
+import RelationshipPicker from './RelationshipPicker';
+import NamePicker from './NamePicker';
+import SearchResultsTable from './SearchResultsTable';
 import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
+import SearchResultsLocation from './SearchResultsLocations';
+import SearchResultsCategory from './SearchResultsCategory';
 
 export default function SearchResults(): JSX.Element {
   const classes = useStyles();
+
   return (
-    <Container className={classes.paper}>
-      <Typography>Hello World</Typography>
+    <Container component="main" className={classes.container}>
+      <Grid container spacing={2}>
+        <Grid item xs={4}>
+          <form noValidate>
+            <Typography className={classes.paper}>Refine your search</Typography>
+            <NamePicker />
+            <LifeEventPicker />
+            <RelationshipPicker />
+            <SearchResultsLocation />
+            <SearchResultsCategory />
+            <div style={{ marginTop: 50 }}>
```

Comment:
I just discovered the Material-UI Box component, which does this in a responsive manner and you don't have to use 'style'. I don't know if it will help here, but you might take a look at it.

Reply:
Will look into it and apply, thanks!

Reply:
Have wrapped the Box component around the Button, thanks!